### DISCUSSION TITLE
Is security vulnerability CVE-2023-45853 fixed in version 4.0.3?

### DISCUSSION THREAD

**Discussion by matthewferler** [nmoinvaz/minizip#748] [General]
A partial description of CVE-2023-45853 is: MiniZip has an integer overflow and resultant heap-based buffer overflow in zipOpenNewFileInZip4_64 via a long filename, comment, or extra field.  I'm wondering if this has been fixed or is on a roadmap to be fixed.  Thanks.

**Comment by pmqs** [comment]
See https://github.com/zlib-ng/minizip-ng/issues/735

**Reply by matthewferler** [reply]
Thank you!

**Reply by Coeur** [reply]
Fixed in #743 (version 4.0.4).
